internal/jobcontainers/storage.go

```diff
@@ -14,10 +15,17 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Trailing backslash required for SetVolumeMountPoint and DeleteVolumeMountPoint
-const sandboxMountFormat = `C:\C\%s\`
+// fallbackRootfsFormat is the fallback location for the rootfs if file binding support isn't available.
+// %s will be expanded with the container ID. Trailing backslash required for SetVolumeMountPoint and
+// DeleteVolumeMountPoint
+const fallbackRootfsFormat = `C:\C\%s\`
```

Comment:
Should this be `C:\hpc\%s`?

Reply:
Yep! Same as above. We decided on the naming after I'd made this and haven't changed yet

Reply:
Updated